rns_shell/platform/graphics/gl/egl/GLWindowContextEGL.cpp

```diff
@@ -438,10 +452,20 @@ std::vector<EGLint> GLWindowContextEGL::RectsToInts(EGLDisplay display, EGLSurfa
     return res;
 }
 
+#if USE(RNS_SHELL_PARTIAL_UPDATES)
 bool GLWindowContextEGL::onHasSwapBuffersWithDamage() {
     return !!eglSwapBuffersWithDamage;
 }
 
+bool GLWindowContextEGL::onHasBufferCopy() {
+#if USE(RNS_SHELL_PARTIAL_UPDATES) && USE(RNS_SHELL_COPY_BUFFERS)
```

Comment:
RNS_SHELL_PARTIAL_UPDATES check seems to be already done at line 455.

Reply:
Addressed